### PR TITLE
Remove devnet references

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

removed the remaining DevNet focused tutorials and updates images

## Links to any relevant issues

fixes https://github.com/iotaledger/devx/issues/325

## Type of change

- Documentation Update
 
## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by eike-hass — APPROVED** [review]

File: contentPlugins.js
Comment (Dr-Electron) [inline contentPlugins.js:1]: They told us the want to update it. So let’s keep it for a little longer

File: versionedConfig.js
Comment (Dr-Electron) [inline versionedConfig.js:1]: Please remove the path require

**Review by Dr-Electron — COMMENTED** [review]
The zebra devs told us their docs will get updated. So we should keep them. And with Identity we agreed to keep 0.6 until 1.1 is ready which should be this coming week maybe?

**Comment by Dr-Electron** [comment]
I see eike agreed to removing the identity docs. So then ignore that part 😆

File: contentPlugins.js
Comment (lucas-tortora) [reply contentPlugins.js:1]: Git doesn't forget; we can always restore it.

**Comment by lucas-tortora** [comment]
> The zebra devs told us their docs will get updated. So we should keep them. And with Identity we agreed to keep 0.6 until 1.1 is ready which should be this coming week maybe?

Already checked with @eike-hass , it's deprecated, and runs on Chrysalis if I'm not mistaken, why keep it?

File: contentPlugins.js
Comment (lucas-tortora) [reply contentPlugins.js:1]: anyhow, middle ground. the files are still there, but I removed indexing and made them partials so they aren't rendered either

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**